cblite/EnrichCommand.cc

```diff
@@ -45,8 +48,20 @@ void EnrichCommand::runSubcommand() {
     processFlags({
         {"--offset", [&]{offsetFlag();}},
         {"--limit",  [&]{limitFlag();}},
+        {"--model",  [&]{_modelName = nextArg("AI Model");}},
     });
     openWriteableDatabaseFromNextArg();
+    
+    Model* model = nullptr;
```

Comment:
This is not good memory management practice.  This pointer is never deleted.  You should make use of either `std::unique_ptr` or `std::shared_ptr`